src/ProjectNeon/Assets/Data/Templates/Deck.cs

```diff
@@ -5,14 +5,12 @@ public class Deck : ScriptableObject
 {
     [SerializeField]
     private List<Card> cards;
-
-    public Deck(List<Card> cards)
-    {
-        this.cards = cards;
-    }
-
     public void Shuffle()
     {
+        /**
+         * @todo #54:30min We should compose deck shuffling behavior. Create a script 
```

Comment:
This isn't needed anymore. All CardPlayZones have shuffling functionality. They are invoked correctly in the setup steps.

Reply:
@SilasReinagel Removed